pyseq.py

```diff
@@ -592,6 +620,116 @@ def _get_missing(self):
 
         return missing
 
+class multiViewSequence(Sequence):
+
+    def __init__(self, items, **kwargs):
+        """
+        multiViewSequence
+        sequence container 
+        items are items from a sequence object egg from a found left view seq[:]
+        kwargs get set as an attribute for each key 
+        {'left': sequence object left, 'right': sequence object}
+        ==> self.left = sequence object left
+            self.right = sequence.object right
+        to iterate through there is a views list
+        for view in seq.views:
+            viewSeq = getattr(seq,view)
+        all format functions on seq only return a %V or %v in the head or tail, depending on the view position
+        """
+        
+        super(multiViewSequence, self).__init__([Item(items.pop(0))])
+
+        while items:
+            f = Item(items.pop(0))
+            try:
+                self.append(f)
+                log.debug('+Item belongs to sequence.')
+            except SequenceError:
+                log.debug('-Item does not belong to sequence.')
+                continue
+            except KeyboardInterrupt:
+                log.info("Stopping.")
+                break    
+        
+        for key in kwargs:
+            setattr(self, key, kwargs[key])
```

Comment:
jep 
one can validate that it needs to be out of the flattened list of stereo pairs....